File: beginner/javascript/1099.js
var input = require('fs').readFileSync('stdin', 'utf8');

var linhas = input.split('\n');
var i, j, n, n1, n2, soma, maior = 0, menor = 0;

n = parseInt(linhas[0]);

for(i = 1; i <= n; i++){
    soma = 0;
    var valores = linhas[i].split(' ').map(parseFloat);

    if(valores[0] < valores[1]){
        menor = valores[0]+1;
        maior = valores[1];
    } else{
        menor = valores[1]+1;
        maior = valores[0];
    }

    for(j = menor; j < maior; j++){
        if(j % 2 != 0){
            soma += j;
        }
    }

    console.log(soma);
}

File: beginner/python/1099.py
maior = 0
menor = 0
i = 0

n = int(input())

for i in range(i,n+1):
    soma = 0
    n1, n2 = map(int, input().split())

    if(n1 < n2):
        menor = n1 + 1
        maior = n2
    else:
        menor = n2 + 1
        maior = n1

    for j in range(menor,maior):
        if(j % 2 != 0):
            soma += j

    print("%d" % soma)